webapp/packages/plugin-data-viewer/src/ContainerDataSource.ts

```diff
@@ -170,6 +169,7 @@ export class ContainerDataSource extends DatabaseDataSource<IDataContainerOption
   private async ensureContextCreated(): Promise<IExecutionContext> {
     if (!this.executionContext) {
       if (!this.options) {
+        this.currentFetchTableProcess = null;
```

Comment:
seems like the wrong place for that, maybe you should do it in `finally` after the process finished